apps/easy-messe/components/OfferMass/LetOfferMass.tsx

```diff
@@ -0,0 +1,343 @@
+                },
+            ]
+        },
+        {
+            name: 'Saint Dominique Savio',
+            city: 'Yaoundé',
+            massData: [
+                {
+                    price: 3000,
+                    dateTime: new Date('2024-06-10T08:30:00'),
+                    massType: null
+                },
+                {
+                    price: 3000,
+                    dateTime: new Date('2024-06-12T09:00:00'),
+                    massType: null
+                },
+                {
+                    price: 3000,
+                    dateTime: new Date('2024-06-16T17:00:00'),
+                    massType: null
+                },
+            ]
+        },
+    ]
+    const massLoop: MassGroupCategory[] = [
+        {
+            label: 'Triduum',
+            valueLoop: 3
+        },
+        {
+            label: 'Seven',
+            valueLoop: 7
+        },
+        {
+            label: 'Novena',
+            valueLoop: 9
+        },
+        {
+            label: 'Thirty',
+            valueLoop: 30
+        },
+    ]
+
+    const inputOfferData: inputProps[] = [
+        {
+            placeholder: 'FullName',
+            icon: editIcon
+        },
+        {
+            placeholder: 'PhoneNumber',
+            icon: contactIcon
+        },
+    ]
+
+    useEffect(() => {
+        setParishDataFetch(parishDataFetched);
+    }, [])
+
+    const matchingParishToCity = parishDataFetched.filter((parish) => parish.city === selectedCity)
+    return (
+        <Box sx={{
+            padding: '21px'
+        }}>
+            <Box sx={{
+                padding: '10px 0 20px 0',
+                display: { laptop: 'grid', mobile: 'none' },
+                gridAutoFlow: 'column',
+                columnGap: 2,
+                width: 'fit-content'
+            }}>
+                {massLoop.map(({ label, valueLoop }, index) => (
+                    <Button
+                        key={index}
+                        variant="outlined"
+                    >
+                        {formatMessage({ id: label })}
+                    </Button>
+                ))}
+            </Box>
+            <Box sx={{
+                display: { laptop: 'none', mobile: 'flex' },
+                flexWrap: 'wrap',
+                columnGap: '40px',
+                rowGap: '10px',
+                paddingBottom: '20px'
+            }}>
+                {massLoop.map(({ label, valueLoop }, index) => (
+                    <Button
+                        key={index}
+                        variant="outlined"
+                        sx={{
+                            minWidth: { laptop: 'initial', mobile: '145px' }
+                        }}
+                    >
+                        {formatMessage({ id: label })}
+                    </Button>
+                ))}
+            </Box>
+            <Typography variant="h2">{formatMessage({ id: 'ApplicantInformation' })}</Typography>
```

Comment:
```suggestion
            <Typography variant="h2">{formatMessage({ id: 'applicantInformation' })}</Typography>
```

these data should be in camelCase